channel/src/main/java/com/synopsys/integration/alert/channel/jira/cloud/JiraCloudContextBuilder.java

```diff
@@ -78,4 +83,23 @@ public JiraCloudContext build(FieldUtility fieldUtility) {
         return new JiraCloudContext(jiraCloudPropertiesFactory.createJiraProperties(fieldUtility), createIssueConfig(fieldUtility));
     }
 
+    @Override
+    public JiraCloudContext build(ConfigurationModel globalConfig, DistributionJobModel jobModel) {
+        FieldUtility globalFieldUtility = new FieldUtility(globalConfig.getCopyOfKeyToFieldMap());
+        JiraCloudProperties jiraProperties = jiraCloudPropertiesFactory.createJiraProperties(globalFieldUtility);
```

Comment:
Is including the properties from `FieldUtlity` for temporary compatibility, or are there fields the context needs that aren't in the `JiraCloudJobDetailsModel`?

Reply:
The former. The Jira channels share several common interfaces and I was trying to simplify for compatibility.